charts/traction-tenant-traceability-controller/templates/controller/environment.yaml

```diff
@@ -5,10 +5,10 @@ metadata:
   name: {{ include "controller.fullname" . }}-env
 type: Opaque
 stringData:
-  POSTGRES_URI: postgres://{{ .Values.postgresql.auth.username }}:{{ .Values.postgresql.auth.secretKeys.userPasswordKey }}@{{ include "global.postgresql.fullname" . }}:{{ .Values.postgresql.primary.service.ports.postgresql }}
```

Comment:
Correct me if I'm wrong but won't this only provision the password key label as a value and not the password itself?

Reply:
This is the current value and does not provision the password value
https://github.com/bcgov/traction-tenant-traceability-controller/blob/b5fc794e0ffafb9b9085c2e7663b804e25242ace/charts/traction-tenant-traceability-controller/templates/controller/environment.yaml#L8